### PR TITLE
Create QR code to insert event for user calendar

### PR DESCRIPTION
**Pull request by chenyang14**

### CODE REVIEW

**Comment by chenyang14** [comment]
Travis-ci runs failed with the error `ImportError: No module named 'qrcode'`
I hve no idea how to solve it, since it runs well on my desktop.

File: arrange_schedule.py [hunk 1]
```diff
@@ -795,6 +797,22 @@ def check_event_exist_or_insert(event):
         }
         with open(receive_msg["text_system_dir"],"w") as fp:
             print(json.dumps(text_file),file=fp)
+        add_event_by_qrcode(event)
+
+def add_event_by_qrcode(eventInfo):
+    path = 'static/calendar_event'
```
Comment (Billy4195) [inline arrange_schedule.py]: Please use `os.path.join('static','calendar_event')`

File: arrange_schedule.py [hunk 2]
```diff
@@ -795,6 +797,22 @@ def check_event_exist_or_insert(event):
...
+    if not os.path.exists(path):
+        os.makedirs(path)
+    startDate = datetime.datetime.strptime(eventInfo['start']['date'], '%Y-%m-%d').strftime('%Y%m%d')
+    endDate = datetime.datetime.strptime(eventInfo['end']['date'], '%Y-%m-%d').strftime('%Y%m%d')
+    if eventInfo['start']['date'] and eventInfo['end']['date']:
+        startTime = '0000'
+        endTime = '0000'
+    #TODO if using start/end 'datetime'
```
Comment (Billy4195) [inline arrange_schedule.py]: Sorry, I can't understand what the TODO want to do.

**Comment by chenyang14** [comment]
@Billy4195  Fixed. Sorry for the poor explanation of `#TODO`. Since that part isn't use at this stage, I just want to keep it for TODO .

**Comment by Billy4195** [comment]
@chenyang14 you said that want to keep it, but you deleted it. 
You can leave TODO for future implementation, but the TODO message should describe what it will be in the future.

**Comment by chenyang14** [comment]
@Billy4195  I add in my latest commit, instead of deleting it.

**Pull request closed without merging**